testcases/include/regress.h

```diff
@@ -117,7 +119,8 @@ int get_user_pin(CK_BYTE_PTR);
 
 #define testcase_setup(total)   \
     t_total = 0;                \
-	t_errors = 0;
+    t_errors = 0;               \
+    gettimeofday(&timev3, NULL);
```

Comment:
When you add a second statement,  please add do {...} while(0) for this function-like macro.

Reply:
Fixed.